.changeset/seven-ghosts-try.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/container-loader": major
+"@fluid-experimental/tree": major
+---
+
+Containers will connect in read-mode by default
```

Comment:
Was this announced in previous majors? If not, should we consider doing that before making this switch? I don't know if our customers are reliant on this and if things may break for them. We should probably add a note saying if this causes issues to contact us and we can help.

Reply:
It was not announced. I will add a note in `main`.

Reply:
I don't think we are supposed to announce in main and then change in next. We have had issues in the past with this and changes had to be backed out. Just making sure that we don't have to do that again :)

Reply:
We have a feature gate, it should be fine..